build-image/Dockerfile

```diff
@@ -5,13 +5,17 @@ MAINTAINER Knut Ahlers <knut@ahlers.me>
 VOLUME /artifacts
 
 RUN apt-get update && \
-    apt-get install -y openssh-client rsync zip wget && \
+    apt-get install -y openssh-client rsync zip wget gnupg && \
     go get github.com/Luzifer/gobuilder/cmd/configreader
 
 ADD ./builder.sh /usr/bin/builder.sh
+ADD ./gpgkey.asc.enc /root/gpgkey.asc.enc
 
 RUN mkdir /root/.ssh && \
     echo "Host *\n\tStrictHostKeyChecking no\n" >> ~/.ssh/config && \
-    chmod 700 /root/.ssh
+    chmod 700 /root/.ssh && \
+    gpg --list-keys 2>&1 1>/dev/null && \
```

Comment:
This just initializes the `/root/` keyring and creates the configs which are modified below.